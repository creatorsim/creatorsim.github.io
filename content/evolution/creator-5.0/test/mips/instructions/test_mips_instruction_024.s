
#
# Creator (https://creatorsim.github.io/creator/)
#

.text
main:

	li $t0, 8191
    li $t1, 8192
    li $t2, 8193
    li $t3, 8194

    slt $t4, $t1, $t0
    slt $t5, $t1, $t1
    slt $t6, $t1, $t2
    slt $t7, $t1, $t3
    
