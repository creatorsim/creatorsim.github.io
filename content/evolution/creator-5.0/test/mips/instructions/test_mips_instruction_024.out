
cr[PC]:0x20; ir[8,t0]:0x1fff; ir[9,t1]:0x2000; ir[10,t2]:0x2001; ir[11,t3]:0x2002; ir[14,t6]:0x1; ir[15,t7]:0x1; keyboard[0x0]:''; display[0x0]:''; 

